app/build.gradle

```diff
@@ -128,12 +128,12 @@ dependencies {
     implementation "androidx.fragment:fragment:$fragmentVersion"
     implementation 'androidx.gridlayout:gridlayout:1.0.0'
     implementation "androidx.media:media:$mediaVersion"
+    implementation "androidx.palette:palette:$paletteVersion"
```

Comment:
How about also doing this for ViewPager and RecyclerView? As soon as we start decoupling the UI, we will need to specify those in different files, too.

Reply:
Good idea 👍

Reply:
Done.